unstract/tool-sandbox/src/unstract/tool_sandbox/helper.py

```diff
@@ -52,7 +52,7 @@ def make_get_request(
         """
         url = f"{self.base_url}{endpoint}"
         params = {"image_name": image_name, "image_tag": image_tag}
-        response = requests.get(url, params=params)
+        response = requests.get(url, params=params, timeout=60)
```

Comment:
Here It might get issue because, some tools need to be run more than 60 seconds. It's better to do with some env variables,